BoardGameProject/src/main/java/com/team4/model/classes/Hand.java

```diff
@@ -3,38 +3,63 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import javax.persistence.CascadeType;
+
 import javax.persistence.Entity;
-import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.OneToMany;
 
 
+
+import javax.persistence.*;
+
 @Entity
+@Table(name = "hand")
 public class Hand {
-	
-
 
 	@Id
-	@GeneratedValue
-	private Long handID;
-	
-	@OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.MERGE)
-	private List<Card> cards;
-	
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@OneToMany
+	private List<Card> cards = new ArrayList<>();
+
 	public Hand() {
-		cards = new ArrayList<>();
+		this.cards = new ArrayList<Card>();
+	}
+
+	public void addCard(Card card) {
+		this.cards.add(card);
 	}
-	
-	public Hand(List<Card> cards) {
-		this.cards = cards;
+
+	public void removeCard(Card card) {
+		this.cards.remove(card);
+	}
+
+	public boolean hasCard(Card card) {
+		return this.cards.contains(card);
```

Comment:
this is never going to return the card you are looking for because the card class's equals() method is not overridden so it will compare the cards by memory address.